src/legacy/ui/public/vis/map/service_settings.js

```diff
@@ -53,6 +53,7 @@ uiModules
           tileApiUrl: mapConfig.emsTileApiUrl,
           htmlSanitizer: $sanitize,
           landingPageUrl: mapConfig.emsLandingPageUrl,
+          fetchFunction: fetch,
```

Comment:
not sure if this is the reason CI is failing, but in Maps, this is wrapped in a lambda. I think there was a specific reason for this, related to compilation, but details escape me now.

https://github.com/elastic/kibana/blob/928454afa4ce0f7136c90b2a8756e0600b55e360/x-pack/legacy/plugins/maps/public/meta.js#L36-L38

Reply:
That appears to have been it. Works perfectly now, thanks!

![image](https://user-images.githubusercontent.com/1728976/77961739-88dc7980-7297-11ea-8eb5-b413e2f06e34.png)